cmd/bosun/sched/check.go

```diff
@@ -198,6 +198,10 @@ func (s *Schedule) CheckUnknown() {
 				continue
 			}
 			a := s.Conf.Alerts[ak.Name()]
+			//alert doesn't exist anymore. Maybe status loaded from old db state.
```

Comment:
There should be no case where an alert doesn't exist where we still have a state. The RestoreState method should take care of those cases. I think that this PR is an correct workaround, but I'd like a better fix.